main/tclx/APKBUILD

```diff
@@ -0,0 +1,47 @@
+# Contributor: Shannon Noe <snoe925@gmail.com>
+# Maintainer: Shannon Noe <snoe925@gmail.com>
+pkgname=tclx
+pkgver=8.4.1
+pkgrel=0
+pkgdesc="TclX extension to Tcl"
+url="http://tclx.sourceforge.net/"
+arch="all"
+license="BSD"
+depends="tcl"
+depends_dev="tcl-dev"
+makedepends="tcl-dev"
+subpackages="$pkgname-doc $pkgname-dev"
+source="http://downloads.sourceforge.net/project/tclx/TclX/$pkgver/$pkgname$pkgver.tar.bz2
+	"
+
+_major=${pkgver%.*}
+_builddir="$srcdir"/tclx${_major}
```

Comment:
Rename this to `builddir` this is now used instead. It also enables e.g. default_prepare to apply patches listed in $source.